files/magneticmediajs/info.ini

```diff
@@ -2,4 +2,4 @@ author = "Arnaud Leyder"
 github = "https://github.com/arnaudleyder/magneticmediajs"
 homepage = "https://www.magneticmediajs.com/"
 description = "JavaScript And CSS To Nicely Display Media Content"
-mainfile = "mjs.min.js"
\ No newline at end of file
+mainfile = "js/mjs.min.js"
```

Comment:
ok just to confirm:
This pull request has 2 distinct issues:
- radiant-medialyzer is a new project I would like to add to jsDelivr. The version is 1.1.0. 
- magneticmediajs is another project already on jsDelivr.  js/mjs.min.js will indeed not exist for 1.3.x and earlier but will exist for 1.4.0 and above. I had to make the change to unify the folder structure with Bower. I was under the impression I had to modify the info.ini but it may not be needed. 

In my opinion putting .js files in the root directory is good for any js only project (which is the case for radiant-medialyzer). For project with css or fonts it is common to use folders like on [bootstrap cdn](http://www.bootstrapcdn.com) with js/ and css/ in the path. This is why I made the change because the file are checkout with the js/ and css/ folder structure and should be accessed as so on the CDN.

Thanks
Arnaud